projects/ngx-drag-to-select/ng-package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
+  "dest": "../../dist/ngx-drag-to-select",
+  "deleteDestPath": false,
+  "lib": {
+    "entryFile": "src/public_api.ts",
+    "styleIncludePaths": ["./src/scss"]
```

Comment:
I tried to build the styles with the cli, but the styles didn't get picked up.
Any ideas? Or do we keep it as it is?

Reply:
I'll check out the PR later on my local machine and see if I find something.

Reply:
You should also add ` "styleIncludePaths": ["./src/scss"]` to `ng-package-prod.json`(?) file because we're building it with the `prod` flag.